src/Veldrid/D3D11/D3D11CommandList.cs

```diff
@@ -738,7 +738,7 @@ private void bindTextureView(D3D11TextureView texView, int slot, ShaderStages st
                 else
                     bind = true;
 
-                if (bind) DeviceContext.PSSetShaderResource(slot, srv);
+                if (bind) DeviceContext.PSSetShaderResource(slot, srv!);
```

Comment:
This looks to be an incorrect NRT in Vortice.Windows. `null` is actually valid here, and the other cases below.